inference-engine/samples/common/utils/src/args_helper.cpp

```diff
@@ -326,3 +359,138 @@ void printInputAndOutputsInfo(const InferenceEngine::CNNNetwork& network) {
                   << layer.second->getLayout() << std::endl;
     }
 }
+void configurePrePostProcessing(std::shared_ptr<ov::Function>& function,
+                                const std::string& ip,
+                                const std::string& op,
+                                const std::string& iop,
+                                const std::string& il,
+                                const std::string& ol,
+                                const std::string& iol,
+                                const std::string& inl,
+                                const std::string& onl,
+                                const std::string& ionl) {
+    auto preprocessor = ov::preprocess::PrePostProcessor();
+    std::unordered_map<size_t, ov::preprocess::InputInfo> inputs_info;
+    std::unordered_map<size_t, ov::preprocess::OutputInfo> outputs_info;
+    const auto inputs = function->inputs();
+    const auto outputs = function->outputs();
+    if (!ip.empty()) {
+        auto type = getType(ip);
+        for (size_t i = 0; i < inputs.size(); i++) {
+            get_info(inputs_info, i).tensor(ov::preprocess::InputTensorInfo().set_element_type(type));
+        }
+    }
+
+    if (!op.empty()) {
+        auto type = getType(op);
+        for (size_t i = 0; i < outputs.size(); i++) {
+            get_info(outputs_info, i).tensor(ov::preprocess::OutputTensorInfo().set_element_type(type));
+        }
+    }
+
+    if (!iop.empty()) {
+        const auto user_precisions_map = parseArgMap(iop);
+        for (auto&& item : user_precisions_map) {
+            const auto& tensor_name = item.first;
+            const auto type = getType(item.second);
+
+            bool isInput = false;
+            for (size_t i = 0; i < inputs.size(); i++) {
+                if (inputs[i].get_names().count(tensor_name)) {
+                    get_info(inputs_info, i).tensor(ov::preprocess::InputTensorInfo().set_element_type(type));
+                    isInput = true;
+                    break;
+                }
+            }
+            if (!isInput) {
+                for (size_t i = 0; i < outputs.size(); i++) {
+                    if (outputs[i].get_names().count(tensor_name)) {
+                        get_info(outputs_info, i).tensor(ov::preprocess::OutputTensorInfo().set_element_type(type));
```

Comment:
what if `tensor_name` is not used?

Reply:
As before we just ignore this parameter.
I can add a logic to throw an exception

Reply:
I think we should throw an exception because user migrating from old to new API can use operation names with new API and preprocessing will not be applied in this case. Users can be confused